metadata-ingestion/src/datahub/ingestion/api/common.py

```diff
@@ -84,3 +91,39 @@ def require_graph(self, operation: Optional[str] = None) -> DataHubGraph:
                 "To provide one, either use the datahub-rest sink or set the top-level datahub_api config in the recipe."
             )
         return self.graph
+
+
+class FlagsConfig(ConfigModel):
```

Comment:
this move shouldn't be fully necessary

we can hide the import with an `if TYPE_CHECKING` block, and then put the import within the `def flags(...)` method

Reply:
Ok I revert moving it and used the `if TYPE_CHECKING` trick,.